test/testenv/testenv.go

```diff
@@ -131,6 +134,11 @@ func (testenv *TestEnv) GetKubeClient() client.Client {
 	return testenv.kubeClient
 }
 
+// UpdateKubeClient returns the kube client to talk to kube-apiserver
+func (testenv *TestEnv) UpdateKubeClient() client.Client {
```

Comment:
Do we need this ? GetKubeClient() is already returning the kubeClient ?

Reply:
Modified